releasenotes-builder/src/main/java/com/github/checkstyle/github/CommitMessage.java

```diff
@@ -61,13 +61,13 @@ public String getMessage() {
 
     /**
      * Checks whether commits message is associated with a pull request or an issue.
-     * Commit message which is associated with a pull request or an issue starts with 'Pull'
-     * or 'Issue' prefix.
+     * Commit message which is associated with a pull request or an issue starts with 'Pull #'
+     * or 'Issue #' prefix.
      *
      * @return true if commits message is associated with a pull request or an issue.
      */
     public boolean isIssueOrPull() {
-        return message.startsWith("Issue") || message.startsWith("Pull");
+        return message.startsWith("Issue #") || message.startsWith("Pull #");
```

Comment:
@romani Confirm this is what you want.

Reply:
This is not what I meant.
Typo can be any. I would rather wrap whole code in try catch and fallback to misc on exception. And log warn in logs that something weird happened to let maintainers to look, and makes decisions on what to do with it.

Reply:
Should be done. Did some method extraction since the indentation would become too much to bear.